scripts/generateAssetData/generateColorMap.ts

```diff
@@ -40,6 +42,7 @@ const generateColorMap = async () => {
     ...optimismAssets,
     ...bnbsmartchainAssets,
     ...polygonAssets,
+    ...gnosisAssets,
```

Comment:
now that it is added to the color map generation, you should run the color map generation and then re run the generate assets script.

Reply:
we should also probably add a `generate:color-map` script to the top level `package.json` to make that step easier

Reply:
taking a look at this one

Reply:
https://github.com/shapeshift/web/pull/4482/commits/172f4d060feacccbcf8496040968508503ff40ce